waf.go

```diff
@@ -39,14 +39,23 @@ type WAFRule struct {
 	AllowedModes []string `json:"allowed_modes"`
 }
 
-// WAFRulesResponse represents the response from the WAF rule endpoint.
+// WAFRulesResponse represents the response from the WAF rules endpoint.
 type WAFRulesResponse struct {
 	Response
 	Result     []WAFRule  `json:"result"`
 	ResultInfo ResultInfo `json:"result_info"`
 }
 
+// WAFRuleResponse represents the response from the WAF rule endpoint.
+type WAFRuleResponse struct {
```

Comment:
In some other examples, the pattern as been to name the singular resource (in this case a WAF rule) as such but then the multiple with `List` in the name. For instance, [`AccessRuleResponse `](https://github.com/cloudflare/cloudflare-go/blob/master/firewall.go#L41-L45) (singular) and [`AccessRuleListResponse `](https://github.com/cloudflare/cloudflare-go/blob/master/firewall.go#L49-L53) (multiple in the response). Do you think it's worth updating here and version bumping to signify a change to the `cloudflare-go` API?

Reply:
I've made the change 👍 note that this will be a breaking change

Reply:
While I am fully sympathetic to the rename, I would recommend against it at this point. We'll have a better chance in the future when we'll do more substantial code cleanups.

Please keep `WAFRulesResponse` for now.